node/_tools/test/parallel/test-fs-read-stream-pos.js

```diff
@@ -79,12 +83,17 @@ const exitTest = () => {
   clearInterval(readInterval);
   clearInterval(writeInterval);
   clearTimeout(endTimer);
-  if (stream && !stream.destroyed) {
-    stream.on('close', () => {
+
+  setTimeout(() => {
+    if (stream && !stream.destroyed) {
+      stream.on('close', () => {
+        assert(cur);
+        process.exit();
+      });
+      stream.destroy();
+    } else {
+      assert(cur);
       process.exit();
-    });
-    stream.destroy();
-  } else {
-    process.exit();
-  }
+    }
+  }, 20);
```

Comment:
A small wait time has been added because common.mustCallAtLeast() validation may also fail if stream.destroy() is called and the stream is terminated in the middle.
And `assert(cur)` has been added to detect a case where the test terminates without ever reading any data as a failure.